src/templates/next/render/renderPageContent.ts

```diff
@@ -0,0 +1,24 @@
+import { renderComponent } from "~/templates/next/render"
+import { IsomerComponent } from "~/types"
+
+export const renderPageContent = ({
+  content,
+  LinkComponent,
+}: {
+  content: IsomerComponent[]
+  LinkComponent: any
+}) => {
+  let isTextOnRight = false
```

Comment:
nit: I think we can make the variable name a little bit clearer? Cos I think currently this is more specific for the alignment of the infopic.

Reply:
Changed in https://github.com/isomerpages/isomer-components/pull/84/commits/4fe54d7bb4390cd14064afe250617388ed13fd50!